app/views/admin/editions/_edition_view_edit_buttons.html.erb

```diff
@@ -18,6 +18,35 @@
       <%= content_data_button(@edition) %>
   <% end %>
 </section>
+
+<% if @edition.state == "draft" %>
```

Comment:
We need a couple more conditions here to display it:
* only in the case of the edition types that we're enabling it for (so not publications etc that have html attachments or links to other documents - details on the ticket)
* only when a user has the permission - Becca P and I looked into this today, so can talk you through it, shouldn't be too bad

Maybe best to put this logic in the controller under a single 'can view preview?' method or something?